src/thermo/critProperties.xml

```diff
@@ -0,0 +1,6331 @@
+<!-- Database of critical properties
+ 
+ These critical properties are used to calculate the a and b parameters 
+ for real gas equations of state, such as Redlich-Kwong and Peng-Robinson.
+ 
+ The a and b parameters can be provided explicitly by the user in the input
+ file.  In the case that they are not provided for a species, the function
+ initThermoXML will search this database for critical properties, which can
+ also be used to calculate the a and b coefficients.  If critical properties
+ are not found and coefficients are not provided, then an error is thrown.
+ 
+ These critical properties have been compiled from various sources by
+ C. Karakaya (https://orcid.org/0000-0001-5102-5476)    
+ -->
+
+<ctml>
```

Comment:
This file in the `src/thermo` directory isn't actually used -- the only one that will get included with an installed copy of Cantera is the one in `data/inputs`. This one seems to have more species, but I'm not sure which one is preferred. In any case, there should only be one file added to the repository for this.